### DISCUSSION TITLE
Remove empty lines that appears when using variables

### DISCUSSION THREAD

**Discussion by Rehvaro** [adieyal/sd-dynamic-prompts#822] [Q&A] [answered]
Hello,

I have a question and wondering if it is resolvable directly in dynamic prompt or if I have to find some nodes to manipulate strings.

If I use a random prompt node with this instructions :

```
${flavor={lemon|strewberry}}
${color={red|blue}}
The cake is ${flavor} flavored and ${color}
```

I get something like :
```


The cake is strewberry flavored and blue
```

Each line that I use for a variable is outputed as an empty line. Is there a way to just ignore these lines in the output and only get :
```
The cake is strewberry flavored and blue
```
without the empty lines ?

Thank you

**Comment by akx** [comment, marked as answer]
> I have to find some nodes to manipulate strings.

implies this is a question related to [the ComfyUI extension](https://github.com/adieyal/comfyui-dynamicprompts), not this stable-diffusion-webui extension.

**Comment by Rehvaro** [comment]
Oh sorry ! Wrong github indeed … 

I will ask there and close the question here. Thank you and sorry again :)
